guides/common/modules/proc_deleting-multiple-content-view-versions.adoc

```diff
@@ -0,0 +1,18 @@
+[id="Deleting_Multiple_Content_View_Versions_{context}"]
+= Deleting Multiple Content View Versions
+
+You can delete multiple content view versions simultaneously.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Content View*.
+. Click the content view you want to delete versions of.
+. Navigate to the *Versions* tab.
+. Select the check box of the version or versions you want to delete. 
+. Click the three dots icon at the top of the list of content views.
+. Click *Delete*.
+. Clicking delete opens the deletion wizard.
+. The deletion wizard displays any affected environments.
```

Comment:
Squash into a single step, remove "clicking delete"

Reply:
We don't document anything except for the user actions. None of the things that the UI does are relevant here. Delete these two steps as they are not user-oriented steps.

Reply:
Ack, Melanie.

Reply:
@AkshayGadhaveRH I don't see this remark incorporated... will you?

Reply:
Ohhh, sorry, my bad.